tools/arriba.wdl

```diff
@@ -0,0 +1,315 @@
+        convert_fusions_to_vcf.sh \
+            ~{fa} \
+            ~{fusions} \
+            ~{vcf}
+    >>>
+
+    output {
+        File fusions_vcf = vcf
+    }
+
+    runtime {
+        cpu: 1
+        memory: "4 GB"
+        disks: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/arriba:2.4.0--h0033a41_2"
+        maxRetries: 1
+    }
+}
+
+task arriba_extract_fusion_supporting_alignments {
+    meta {
+        description: "Extract alignments that support fusions."
+        outputs: {
+            fusion_bams: "Array of BAM files corresponding with fusions in the input file"
+            fusion_bam_indexes: "Array of BAM indexes corresponding with the BAMs in the 'fusion_bams'"
+        }
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file on which to call fusions"
+        bam_index: "BAM index file corresponding to the input BAM"
+        fusions: "Input fusions in TSV format to convert to VCF"
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        File bam
+        File bam_index
+        File fusions
+        String prefix = basename(fusions, ".tsv")
+        Int modify_disk_size_gb = 0
+    }
+
+    Int input_size_gb = ceil(size(bam, "GiB"))
+    Int disk_size_gb = ceil(input_size_gb) + 5 + modify_disk_size_gb
+
+    command <<<
+        extract_fusion-supporting_alignments.sh \
+            ~{fusions} \
+            ~{bam} \
+            ~{prefix}
+    >>>
+
+    output {
+        Array[File] fusion_bams = glob("~{prefix}_*.bam")
+        Array[File] fusion_bam_indexes = glob("~{prefix}_*.bam.bai")
+    }
+
+    runtime {
+        cpu: 1
+        memory: "4 GB"
+        disks: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/arriba:2.4.0--h0033a41_2"
+        maxRetries: 1
+    }
+}
+
+task arriba_annotate_exon_numbers {
+    meta {
+        description: "Annotate fusions with exon numbers."
+        outputs: {
+            fusion_bams: "Array of BAM files corresponding with fusions in the input file"
+            fusion_bam_indexes: "Array of BAM indexes corresponding with the BAMs in the 'fusion_bams'"
+        }
+    }
+
+    parameter_meta {
+        fusions: "Input fusions in TSV format to convert to VCF"
+        gtf: "GTF features file. Gzipped or uncompressed."
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        File fusions
+        File gtf
+        String prefix = basename(fusions, ".tsv") + ".annotated.tsv"
+        Int modify_disk_size_gb = 0
+    }
+
+    Int input_size_gb = ceil(size(gtf, "GiB"))
+    Int disk_size_gb = ceil(input_size_gb) + 5 + modify_disk_size_gb
+
+    String gene_model = basename(gtf, ".gz")
+
+    command <<<
+        if [ "~{gene_model}" != "~{gtf}" ]
```

Comment:
This won't work as expected. `gtf` will have an absolute path, `gene_model` will only have a basename.

Is there a reason you can't copy+paste from one of our other tasks that does this? Using `ln -sf`?